src/SockPerf.cpp

```diff
@@ -3205,6 +3231,16 @@ int bringup(const int *p_daemonize)
 			log_msg("Number of threads should be less than sockets count");
 			rc = SOCKPERF_ERR_BAD_ARGUMENT;
 		}
+
+		if ( !rc && s_user_params.dummy_send && s_user_params.mps == UINT32_MAX) {
+			log_err("Dummy send is not supported when message-rate = MAX");
+			rc = SOCKPERF_ERR_BAD_ARGUMENT;
+		}
+
+		if ( !rc && s_user_params.dummy_send && s_user_params.mps > s_user_params.dummy_mps) {
+			log_err("Dummy send is not supported when message-rate is greater then the dummy-message-rate");
```

Comment:
error msg: "Dummy send is allowed only if dumm-send-rate is lower than regular msg rate"